docker-compose.dev.yml

```
@@ -0,0 +1,15 @@
+version: "3.3"
+services:
+  flight_review_app:
+    build:
+      context: ./app
+      dockerfile: Dockerfile.dev
+    restart: always
+    network_mode: bridge
+    ports:
+     - 5006:5006
+    env_file: .env
+    command: /opt/service/run.sh
+    volumes:
+      - ./app:/opt/service/
+      - ./data/app_data:/opt/service/data
```

I guess this should be:
```suggestion
      - ./data:/opt/service/data
```